### PR TITLE
Nightmarket fest UI change

### PR DESCRIPTION
**Pull request by arch-jslin**
as title

### CODE REVIEW

**Comment by elct9620** [comment]
Please update `features/night_market.feature` to click correct menu text after translation is added.

**Comment by arch-jslin** [comment]
feature request updated

File: features/night_market.feature [hunk 1]
```diff
@@ -49,3 +49,8 @@ Feature: Night Market
     When  I visit "/"
     And I click "Night Market" in menu
     Then I can see the text block with "Hello World"
+    
+  @night_market_enabled
```
Comment (elct9620) [inline features/night_market.feature:53]: The E2E test failed is due to `And I click "Night Market" in menu` should be `And I click "遊戲夜市" in menu` in the scenario. We didn't need to add a test for this scenario.

**Review by elct9620 — CHANGES_REQUESTED** [review]

**Comment by arch-jslin** [comment]
E2E test updated. The final feature test is marked "pending" because there are no feature tests for button translation correctness.

File: features/night_market.feature (hunk 1 above)
Comment (arch-jslin) [reply features/night_market.feature:53]: I think I've fixed all feature tests

**Review by arch-jslin — COMMENTED** [review]
@elct9620 please see if you can approve the review now.

**Pull request merged**